### PR TITLE
INT-574 Refactor calls

### PR DESCRIPTION
**Pull request by BullDog57Rus**

### CODE REVIEW

File: talk/talk-api.yaml [hunk 1]
```diff
@@ -396,6 +396,32 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ResponseError'
+  /webhooks/calls:
+    x-access: private
```
Comment (Onbehalfofme) [inline talk/talk-api.yaml:400]: Add THIS IS INTERNAL ENDPOINT in description for private methods

File: talk/talk-api.yaml [hunk 2]
```diff
@@ -668,8 +694,67 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ResponseError'
+  /calls/{id}/participants:
+    post:
+      tags:
+        - Call Participants
+      summary: Add participant to the call
+      description: Use this endpoint to add a new participant to a call. Provide a call ID in the path.
+      operationId: createParticipant
```
Comment (Onbehalfofme) [inline talk/talk-api.yaml:703]: maybe addParticipant instead of create

File: talk/talk-api.yaml [hunk 3]
```diff
@@ -668,8 +694,67 @@ paths:
...
+              schema:
+                $ref: '#/components/schemas/CallParticipant'
+        default:
+          description: Bad request, security violation, or internal server error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResponseError'
+  /webhooks/calls/{id}/participants/{subId}/hang:
+    x-access: private
```
Comment (Onbehalfofme) [inline talk/talk-api.yaml:727]: Add description

File: talk/talk-api.yaml [hunk 4]
```diff
@@ -668,8 +694,67 @@ paths:
...
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CallParticipant'
+        default:
+          description: Bad request, security violation, or internal server error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResponseError'
+  /webhooks/calls/{id}/participants/{subId}/hang:
```
Comment (Onbehalfofme) [inline talk/talk-api.yaml:726]: `/webhooks/calls/{id}/participants/{userId}/hang:`

File: talk/talk-api.yaml [hunk 5]
```diff
@@ -790,6 +876,7 @@ paths:
                 $ref: '#/components/schemas/ResponseError'
   /calls/callbacks/outbound:
     post:
+      deprecated: true
```
Comment (Onbehalfofme) [inline talk/talk-api.yaml:879]: Maybe it makes sense to make a description like with the internal endpoint. Something like DEPrecated at the beginning

**Review by Onbehalfofme — CHANGES_REQUESTED** [review]

File: talk/talk-api.yaml (hunk 2 above)
Comment (BullDog57Rus) [reply talk/talk-api.yaml:703]: I try to stick to CRUD naming, just to make it understandable for everyone. And addition of the participant is more about the business logic, but not the REST api

**Review by Onbehalfofme — APPROVED** [review]

**Pull request merged**